install.sh

```diff
@@ -1,15 +1,16 @@
 #!/bin/bash
-set -x
-echo "update phase"
+set -e
```

Comment:
Some code will definitely failed.
ex. ```apt-get remove -y nginx``` or ```apt-get remove -y npm```
If you installed it on new Ubuntu, ```nginx``` will not exist.
But if you installed in existing Ubuntu, ```nginx``` are conflict with ```nginx-full``` (Some times..... at least in my debian, it happend)
and
```
usermod -aG shadow nginx
usermod -aG shadow www-data
```
so on, I wrote reason below.